src/com/massivecraft/massivetickets/cmd/CmdTicketsCreate.java

```diff
@@ -61,6 +62,12 @@ public void perform() throws MassiveException
 		boolean update = msender.hasMessage();
 		String verb = update ? "updated" : "created";
 		
+		// But only if the ticket has been changed
+		if (update && MUtil.equals(msender.getMessage(), message))
+		{
+			msg("<b>Please do not spam our ticket system. Someone will be with you shortly.");

```

Comment:
Is this really the best way to do it? It could be seen as a bit passive-agressive, and it would also make people realise that the message hasn't been forwarded which makes it easier to trick the system by changing it ever so slightly.
Perhaps just act normally towards the player and not send a message to the moderators?